src/ResourceTemplate/Segment.php

```diff
@@ -61,6 +61,10 @@ class Segment
     /** @var string */
     private $stringRepr;
 
+    /** @var string */
+    private $separator;
+
+
```

Comment:
This extra whitespace can also be removed.

Reply:
Done.